CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# Unreleased v1.23.0.0.0
+
+## Added
+
+- HandleBuilder class for managing libddwaf configuration and building WAF handles
```

Comment:
1. Will be cool to use code signature to distinguish between text and class names "`"
2. I think we should stick to the format `Add/Update/Remove` as the beginning of each item

Reply:
good point, I think it became more readable: [d25c605](https://github.com/DataDog/libddwaf-rb/pull/77/commits/d25c6050ee22f3e1e570ca63e05002ea19a2062c)